decidim-consultations/spec/scrubbers/decidim/consultations/question_title_scrubber_spec.rb

```diff
@@ -0,0 +1,61 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+describe Decidim::Consultations::QuestionTitleScrubber do
+  subject { described_class.new }
+
+  def scrub(html)
+    Loofah.scrub_fragment(html, subject).to_s
+  end
+
+  RSpec::Matchers.define :be_scrubbed do
+    match do |actual|
+      expect(scrub(actual)).to eq actual
+    end
+
+    failure_message do |actual|
+      "expected \"#{actual}\" to eq \"#{scrub(actual)}\" after scrubbing"
+    end
+  end
+
+  RSpec::Matchers.define :be_scrubbed_as do |expected|
+    match do |actual|
+      expect(scrub(actual)).to eq expected
+    end
+
+    failure_message do |actual|
+      "expected \"#{actual}\" to eq \"#{expected}\" after scrubbing, scrubbed as \"#{scrub(actual)}\" instead"
+    end
+  end
+
+  it "does not allow iframes" do
+    html = "<iframe frameborder=\"0\" allowfullscreen=\"true\" src=\"url\"></iframe>"
+    expect(html).to be_scrubbed_as("")
+  end
+
+  it "does not allow comments" do
+    html = "<p>Hello, <!-- world! --></p>"
+    expect(html).to be_scrubbed_as("<p>Hello, </p>")
+  end
+
+  it "does not allow disabled iframes" do
+    html = %(<div class="disabled-iframe"><!-- <iframe src="url"></iframe> --></div>)
```

Comment:
This will not work on 0.26.

The iframe disabler was introduced by 0.27.0 through the data consent change (search "data consent change" from the release notes):
https://github.com/decidim/decidim/releases/tag/v0.27.0

It requires some JavaScript to function correctly which we do not want to introduce to 0.26 because it's related to the data consent, i.e. a feature that was not available in 0.26.

Prior to 0.27 we used to show the iframes on page (where allowed, i.e. admin entered content) straight away.

Reply:
In 0.26 we do not have the IframeDisabler nor decidim_sanitize_editor_admin method ... We could implement the `decidim_sanitize_editor_admin` just to perform the scrubbing, but we need to make further changes like allow the iframe. 

The alternative would be not to apply the patch and close this PR.

Reply:
Yes, it's a tricky one for sure. I would say let's introduce `decidim_sanitize_editor_admin` in 0.26 and backport this.

IframeDisabler should not be backported. Only the separation for admin and participant scrubbing.